gpkit/constraints/signomial_program.py

```diff
@@ -125,8 +133,23 @@ def localsolve(self, solver=None, verbosity=1, x0=None, rel_tol=1e-4,
         self.result = result  # NOTE: SIDE EFFECTS
         return result
 
+    def _fill_x0(self, x0):
+        "Fills x0 with subsitutions and sp_inits"
```

Comment:
it doesn't mutate the passed-in `x0`, right? Would be more accurate to say "Returns a copy of x0 with substitutions and sp_inits added"?